pom.xml

```diff
@@ -18,11 +18,11 @@
 
   <licenses>
     <license>
-      <name>Eclipse Public License - Version 2.0</name>
+      <name>EPL-2.0</name>
       <url>https://www.eclipse.org/legal/epl-2.0/</url>
     </license>
     <license>
-      <name>Apache Software License - Version 2.0</name>
+      <name>Apache-2.0</name>
       <url>https://www.apache.org/licenses/LICENSE-2.0</url>
```

Comment:
Lets use the same URL as [`org.apache:apache` parent pom](https://github.com/apache/maven-apache-parent/blob/2deae599f76cedeb4b0d083e067377a4941cf95a/pom.xml#L45-L49) too.

``` xml
<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
```